### PR TITLE
fix(ios): save `resizeMode` and reapply it when replacing a view

### PR DESCRIPTION
**Pull request by hardworker**
This PR addresses the following issue: https://github.com/lottie-react-native/lottie-react-native/issues/1114

### CODE REVIEW

File: packages/core/ios/LottieReactNative/ContainerView.swift [hunk 1]
```diff
@@ -296,8 +289,6 @@ class ContainerView: RCTView {
     func replaceAnimationView(next: LottieAnimationView) {
         super.removeReactSubview(animationView)
 
-        let contentMode = animationView?.contentMode ?? .scaleAspectFit
-
         animationView = next
 
         animationView?.contentMode = contentMode
```
Comment (TheRogue76) [inline packages/core/ios/LottieReactNative/ContainerView.swift:294]: If the functionality is being extracted, i suppose this no longer makes any sense to be here

File: packages/core/ios/LottieReactNative/ContainerView.swift [hunk 2]
```diff
@@ -317,6 +309,20 @@ class ContainerView: RCTView {
         }
     }
 
+    func applyContentMode() {
+        guard let animationView else { return }
```
Comment (TheRogue76) [inline packages/core/ios/LottieReactNative/ContainerView.swift:313]: This is newer swift syntax, and may break on older versions (We support Swift 5.6) maybe change it to:
```suggestion
        guard let animationView = animationView else { return }
```
To ensure compatibility

**Comment by TheRogue76** [comment]
Hi @hardworker. Thank you for the contribution. I left two comments, once you had time to address those, please test on new Arch and old arch, and we can get this merged 🙏

**Comment by TheRogue76** [comment]
Hi @hardworker. We are planning on doing a release after this is merged. Please let me know if you can wrap up the fixes in the upcoming week, otherwise i can just do the minor adjustments in this PR and do the tests and we can have this merged and as part of the release 🙏.

**Comment by matinzd** [comment]
Any updates on this?

**Comment by hardworker** [comment]
Hey, @TheRogue76! Sure, I'll do the fixes later today and ping you :)

**Comment by hardworker** [comment]
@TheRogue76 Hello again, I made the necessary changes, added a small fix to address a SwiftLint warning and tested on both Fabric and Paper, works just fine!

**Review by TheRogue76 — APPROVED** [review]
LGTM. @matinzd we are ready

**Pull request merged**